### PR TITLE
chore: Bump faraday and hyperclient dependencies

### PR DESCRIPTION
**Pull request by alexevanczuk**
@nebolsin or @nullstyle 

Would you be able to help me get this merged?

We were hoping to update some of the dependencies so we could use this gem in our app.

Thank you!

### CODE REVIEW

File: horizon/stellar-horizon.gemspec [hunk 1]
```diff
@@ -28,7 +28,7 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency "excon", ">= 0.71.0", "< 1.0"
   spec.add_dependency "faraday", ">= 1.6.0", "< 3.0"
-  spec.add_dependency "faraday-excon", ">= 1.1.0", "< 3.0"
+  spec.add_dependency "faraday-excon", "~> 2.1"
```
Comment (alexevanczuk) [inline horizon/stellar-horizon.gemspec:31]: https://github.com/excon/faraday-excon

File: Gemfile.lock [hunk 2]
```diff
@@ -70,35 +70,15 @@ GEM
...
-      faraday-em_synchrony (~> 1.0)
-      faraday-excon (~> 1.1)
-      faraday-httpclient (~> 1.0)
-      faraday-multipart (~> 1.0)
-      faraday-net_http (~> 1.0)
-      faraday-net_http_persistent (~> 1.0)
-      faraday-patron (~> 1.0)
-      faraday-rack (~> 1.0)
-      faraday-retry (~> 1.0)
+    faraday (2.0.0)
```
Comment (alexevanczuk) [inline Gemfile.lock:73]: https://github.com/lostisland/faraday/blob/main/UPGRADING.md

**Comment by alexevanczuk** [comment]
Hey @nebolsin and @nullstyle , wondering if you're still maintaining this gem and if it's possible for us to approve the CI workflows and maybe get this merged 🙏🏼 

Thank you!

**Comment by alexevanczuk** [comment]
@nebolsin It looks like some of the automated actions might be broken unrelated to this PR, are you seeing the same thing?

**Comment by alexevanczuk** [comment]
Hey team going to close this for now, but please let me know if you're available to help get it merged and I'll reopen it!

**Comment by nebolsin** [comment]
@alexevanczuk Hey, sorry for the late response. Thank you for the PR, upgrading Faraday has been on my radar for quite some time. I'll take a look into the CI failures and try to get this merged.

**Comment by alexevanczuk** [comment]
Amazing ok thank you @nebolsin !!

**Comment by alexevanczuk** [comment]
Hey @nebolsin just wanted to follow up on this!

**Pull request closed without merging**